src/couch_replicator_manager.erl

```diff
@@ -451,7 +487,14 @@ maybe_start_replication(State, DbName, DocId, RepDoc) ->
         true = ets:insert(?DOC_TO_REP, {{DbName, DocId}, RepId}),
         couch_log:notice("Attempting to start replication `~s` (document `~s`).",
             [pp_rep_id(RepId), DocId]),
-        Pid = spawn_link(fun() -> start_replication(Rep, 0) end),
+        StartDelaySecs = erlang:max(0, list_to_integer(
+            config:get("replicator", "start_delay", "10"))),
```

Comment:
There is nicer helper [config:get_integer(Key, Value, Default)](https://github.com/apache/couchdb-config/blob/master/src/config.erl#L60-L66).